book.rb

```diff
@@ -0,0 +1,13 @@
+class Book
+  attr_accessor :title, :author, :rental
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rental = []
```

Comment:
- [ ] Since you pluralized `rentals` in the Rental class, kindly do the same here. This is to prevent errors from showing up when creating a rental.

  ```
  :rentals
  @rentals = [ ]
  ```